jwst/outlier_detection/outlier_detection_step.py

```diff
@@ -58,17 +58,22 @@ class OutlierDetectionStep(Step):
         snr = string(default='5.0 4.0')
         scale = string(default='1.2 0.7')
         backg = float(default=0.0)
-        save_intermediate_results = boolean(default=False)
+        save_intermediate_results = boolean(default=True)
         resample_data = boolean(default=True)
         good_bits = string(default="~DO_NOT_USE")  # DQ flags to allow
         scale_detection = boolean(default=False)
         search_output_file = boolean(default=False)
         allowed_memory = float(default=None)  # Fraction of memory to use for the combined image.
+        in_memory = boolean(default=False)
+        output_use_index = boolean(default=False)
+        output_use_model = boolean(default=False)
+        save_results = boolean(default=True)
```

Comment:
I believe ``save_results`` is a standard param that's hardwired into every `Step` object, so is it necessary to define it here too?

Reply:
Probably not.  It has been removed in commit 351c28d5.